runtime/amplitude/src/lib.rs

```diff
@@ -1251,6 +1262,8 @@ construct_runtime!(
 		VaultRewards: reward::{Pallet, Call, Storage, Event<T>} = 70,
 		VaultStaking: staking::{Pallet, Storage, Event<T>} = 71,
 
+		AssetRegistry: orml_asset_registry::{Pallet, Storage, Call, Event<T>, Config<T>} = 91,
```

Comment:
Maybe it is good to add a comment here in order not to give the impression that this is another Spacewalk pallet (same for the other `construct_runtime!` invocations in the other runtimes.

Reply:
That's fine.